### ISSUE TITLE
No content in the plugin window

### ISSUE THREAD

**Issue by andrewatfanduel** [berzniz/github_pr_tree#98]
For some reason, I'm now seeing no content in the container where all the plugin content normally lives. 

![Screenshot 2020-02-27 at 12 46 05](https://user-images.githubusercontent.com/19692210/75446348-23772f00-595f-11ea-8e38-9c6a7e822e68.png)

Happening on any PR I visit, currently in a private repo console logs from chrome:

```
Uncaught TypeError: Cannot read property 'getElementsByClassName' of null
    at l (better_github_prs.js:1)
    at better_github_prs.js:1
    at Array.forEach (<anonymous>)
    at better_github_prs.js:1
    at Array.forEach (<anonymous>)
    at c (better_github_prs.js:1)
    at n.value (better_github_prs.js:31)
    at n.value (better_github_prs.js:31)
    at Ia (better_github_prs.js:22)
    at Ma (better_github_prs.js:22)
```

Happy to have a look at it, if its an issue not just related to me

**Comment by berzniz** [comment]
Thanks for reporting. This is already solved with #96 -- and will be out to the store (version 1.0.26) once Google approves it. Hopefully today.

**Comment by andrewatfanduel** [comment]
🎆  cheers!
